src/connectedk8s/azext_connectedk8s/_utils.py

```diff
@@ -836,12 +886,15 @@ def which(binary):
 
 def try_archive_log_file(troubleshoot_log_path, output_file):
     try:
+        periscope_zip = os.path.join(os.path.expanduser('~'), '.azure', 'periscope_output.tar.gz')
         # Creating the .tar.gz for logs and deleting the actual log file
         import tarfile
         with tarfile.open(output_file, "w:gz") as tar:
             tar.add(troubleshoot_log_path, 'connected8s_troubleshoot.log')
+            tar.add(periscope_zip, 'periscope_output.tar.gz')
         logging.shutdown()  # To release log file handler, so that the actual log file can be removed after archiving
         os.remove(troubleshoot_log_path)
+        os.remove(periscope_zip)
         print(f"{colorama.Style.BRIGHT}{colorama.Fore.GREEN}Some diagnostic logs have been collected and archived at '{output_file}'.")
```

Comment:
Remove "some" - "Diagnostic logs have been collected and archived at..."


I had kept some earlier since only non-periscope logs was there.